src/curategpt/app/app_alz.py

```diff
@@ -57,15 +56,22 @@
 cart = state.cart
 
 
-st.title("Alzheimers AI assistant")
+st.title("Alzheimer's AI Assistant")
+
+# Check if PQA_HOME environment variable is set for PaperQA
+if PAPERQA in [PUBMED, PAPERQA, WIKIPEDIA] and os.environ.get("PQA_HOME") is None:
```

Comment:
Could this be passed to the CLI instead and/or have a default location (set in the `paperqawrapper` below)?
Oh nevermind, I see that's set with the `index` command through the CLI.